### ISSUE TITLE
Any Way to Install with Toybox Instead of Sbase and Ubase

### ISSUE THREAD

**Issue by VehementHam** [oasislinux/oasis#108]
If so, how, and this is a package request.

**Comment by apprehensions** [comment]
No, Toybox requires a heavily complicated build script (which not only compiles programs, it literally builds the programs code) which uses Bash, and is very difficult to port to POSIX sh.

**Comment by VehementHam** [comment]
What about Busybox
